junit-test/output-native/src/test/scala/scala/scalanative/junit/utils/JUnitTestPlatformImpl.scala

```diff
@@ -7,7 +7,7 @@ import java.nio.charset.StandardCharsets.UTF_8
 
 import sbt.testing._
 
-import scala.collection.JavaConverters._
+import scala.jdk.CollectionConverters._
```

Comment:
That cannot work in `junitTestOutputsNative`. It is a Native project, so we cannot depend on `scala-collection-compat` in it. For the JUnit output tests, in Scala.js we still use the old API, even if it is deprecated in 2.13. I think we can do the same here (and also in `output-jvm` for consistency).